.pylintrc.toml

```diff
@@ -521,7 +522,7 @@ missing-member-max-choices = 1
 mixin-class-rgx = ".*[Mm]ixin"
 
 # List of decorators that change the signature of a decorated function.
-# signature-mutators =
+signature-mutators = ["xclim.sdba.base.map_groups"]
```

Comment:
wouldn't adding "numba.guvectorize" solve the issues in  xclim/indices/fire/_ffdi.py and xclim/sdba/nbutils.py ?

Reply:
For the record, that doesn't work. Maybe one day this will be resolved.